src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -86,11 +112,133 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
     expect((await screen.findAllByRole('radio')).length).to.equal(2);
     expect(screen.getByLabelText('Bozeman, MT')).to.exist;
     expect(screen.getByLabelText('Belgrade, MT')).to.exist;
+    expect(screen.baseElement).to.contain.text(
+      'Request a Primary care provider. (Optional)',
+    );
+    expect(screen.baseElement).to.contain.text(
```

Comment:
I'd probably leave this expect out, we don't really need to verify that this description text is handled correctly by the form system

Reply:
Fixed